### ISSUE TITLE
Allow opting out of the destroy phase for `emberGenerateDestroy`.

### ISSUE THREAD

**Issue by rwjblue** [ember-cli/ember-cli-blueprint-test-helpers#54]
Due to https://github.com/ember-cli/ember-cli/issues/5884, I cannot test destroying my addon's default blueprint. I have had to resort to the following (instead of using `emberGenerateDestroy`):

```js
var emberGenerateDestroy = blueprintHelpers.emberGenerateDestroy;
var chai = require('ember-cli-blueprint-test-helpers/chai');
var expect = chai.expect;
var file = chai.file;

  it('ember-cli-template-lint without localization framework', function() {
    var args = ['ember-cli-template-lint'];

    return emberNew()
      .then(function() {
        return emberGenerate(args)
      })
      .then(function() {
          expect(file('.template-lintrc.js')).to.contain('extends: \'recommended\'');
      });
  });
```

Obviously, we should ensure the upstream issue is fixed and this becomes less of an issue, but should we allow an option to opt out of the destroy phase for now?

**Comment by Turbo87** [comment]
What's the advantage of opting out compared to the solution above?

**Comment by trabus** [comment]
I don't think opting out should be necessary, there shouldn't ever be a situation where a blueprint can be generated but not destroyed (which means we should fix ember-cli/ember-cli#5884 asap). I'd prefer to be explicit in situations where you only want to check generate or destroy but not both.

**Comment by rwjblue** [comment]
Seems good. I can close, and reference this as the solution for posterity/Google/etc...
